nncf/tensorflow/quantization/quantize_model.py

```diff
@@ -126,6 +126,8 @@ def _create_nncf_config(
     if advanced_parameters is not None:
         compression_config = apply_advanced_parameters_to_config(compression_config, advanced_parameters)
 
+    compression_config["strict_check_scopes"] = False
```

Comment:
I would suggest change behavior of `nncf.quantize()` for `torch` and `tensorflow` backends more granularity:
- `if model_type==None then compression_config["strict_check_scopes"] = True`
- `if model_type==ModelType.TRANSFORMER then compression_config["strict_check_scopes"] = False`, because the algorithm specified additional `IgnoredScope`.

In this case we will obtain more align behavior with other backends.

Reply:
Update, "strict_check_scopes"= False only for ModelType.TRANSFORMER.

Reply:
`strict_check_scopes` -> `validate_scopes` if it work for both ignored and target scopes. Otherwise, it is better to call it `validate_ignored_scopes`